docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc

```diff
@@ -1,29 +1,23 @@
 [id="creating-a-ceph-nfs-cluster_{context}"]
 
-= Creating a NFS Ganesha cluster
+= Creating an NFS Ganesha cluster
 
-If you use the Ceph via NFS backend with {rhos_component_storage_file_first_ref}, prior to adoption, you must create a new clustered NFS service on the Ceph cluster. This service will replace the standalone, pacemaker-controlled `ceph-nfs` service that was used on {rhos_prev_long} {rhos_prev_ver}.
+If you use the Ceph via NFS back end with the {rhos_component_storage_file_first_ref}, you must create a new clustered NFS service on the {Ceph} cluster. This service replaces the standalone, Pacemaker-controlled `ceph-nfs` service that you use in {rhos_prev_long} ({OpenStackShort}) {rhos_prev_ver}.
 
 .Procedure
 
-. You must identify the ceph nodes to deploy the new clustered NFS service.This service must be deployed on the `StorageNFS` isolated network so that
-it is easier for clients to mount their existing shares through the new NFS
-export locations.
-. You must propagate the `StorageNFS` network to the target nodes
-where the `ceph-nfs` service will be deployed.
+. Identify the {Ceph} nodes to deploy the new clustered NFS service. You must deploy this service on the `StorageNFS` isolated network so that
+you can mount your existing shares through the new NFS export locations.
```

Comment:
Is there a command to identify the nodes? Or are we just saying that the customer needs to decide which nodes they want to deploy the new NFS service on?

Reply:
@fmount 
Maybe something like the following?
[NOTE]
You can deploy the new clustered NFS service on your existing CephStorage nodes or HCI nodes, or on new hardware that you enrolled in the Ceph cluster.

Reply:
+1 to have a note, thank you!